### PR TITLE
Font Integration Test

### PR DESCRIPTION
**Pull request by melloware**
Just starting to add changes that my code uses. Font was first.

I think the Autosizing has to know the Font to know how big the cells are.

### CODE REVIEW

**Comment by gastaldi** [comment]
Looks like it still works :)

**Comment by melloware** [comment]
Yeah Strange I am trying to figure out what else would trigger the FontManager and have it not work.

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -90,6 +90,8 @@
       <properties>
         <skipITs>false</skipITs>
         <quarkus.package.type>native</quarkus.package.type>
+        <quarkus.native.container-build>true</quarkus.native.container-build>
```
Comment (gastaldi) [inline integration-tests/pom.xml:93]: Yeah, that doesn't work for the integration tests

**Comment by melloware** [comment]
Don't mind me just taking stabs at seeing what is different. 😄

**Comment by gastaldi** [comment]
BTW you can also see these build results if you push to your fork locally, no need to submit a PR for that

**Pull request closed without merging**